### DISCUSSION TITLE
Fine-tune italics settings

### DISCUSSION THREAD

**Discussion by tmummert** [rose-pine/neovim#176] [Q&A] [answered]
Hello,

the option `disable_italics = boolean` switches italics on/off as a whole.

Is it possible to fine-tune this behavior?

This seems to have no effect:
```
styles = {
        comments = { italic = true },
        keywords = { italic = false },
        …
}
```

Thank you very much!

**Comment by mvllow** [comment, marked as answer]
Use our `highlight_groups`:

```lua
highlight_groups = {
  Comment = { italic = true }
}
```

In older versions of our theme the above would overwrite the entire "Comment" group but in the latest version it is extended so make sure you're on the latest version :)

**Reply by tmummert** [reply]
This works perfectly, thank you very much!

**Reply by R4405U** [reply]
how to make Keywords like if,else,for,while,etc italic ?

**Reply by mvllow** [reply]
Try putting your cursor over any of the words you mentioned, e.g. "if" and run `:Inspect`. That should give you a highlight group and then copy the above, replacing "Comment" with the new highlight group.

**Reply by R4405U** [reply]
works perfectly, this helped a lot, thank you very much!

**Reply by anakinsonone** [reply]
@mvllow Thanks for that insight! Really helpful.
